add release notes for 1.25.4 (#16753)
istio/istio.io: content/en/news/releases/1.25.x/announcing-1.25.4/index.md

---
title: Announcing Istio 1.25.4
linktitle: 1.25.4
subtitle: Patch Release
description: Istio 1.25.4 patch release.
publishdate: 2025-08-08
release: 1.25.4
---

This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.25.3 and Istio 1.25.4.

{{< relnote >}}

## Changes

- **Fixed** an issue where Istio upgrade from 1.24 to 1.25 caused service disruption due to preexisting iptables rules.

>     The iptables binary detection logic has been improved to verify a degree of baseline kernel support exists, and prefer `nft` in a `tie` situation.

Author: ```suggestion
    The iptables binary detection logic has been improved to verify a degree of baseline kernel support exists, and prefer `nft` in a `tie` situation.

```
Restore newline without trailing spaces.

---
title: Announcing Istio 1.25.4
linktitle: 1.25.4
subtitle: Patch Release
description: Istio 1.25.4 patch release.
publishdate: 2025-08-08
release: 1.25.4
---

This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.25.3 and Istio 1.25.4.

{{< relnote >}}

## Changes

- **Fixed** an issue where Istio upgrade from 1.24 to 1.25 caused service disruption due to pre-existing iptables rules.
The iptables binary detection logic has been improved to verify a degree of baseline kernel support exists, and prefer `nft` in a `tie` situation.
  
- **Fixed** false positive with `istioctl analyze` raising IST0134 even when `PILOT_ENABLE_IP_AUTOALLOCATE` is set to `true`.
  ([Issue #56083](https://github.com/istio/istio/issues/56083))

- **Fixed** a panic in `istioctl manifest translate` when the IstioOperator config contains multiple gateways.
  ([Issue #56223](https://github.com/istio/istio/issues/56223))

- **Fixed** ambient index to filter configs by revision.
  ([Issue #56477](https://github.com/istio/istio/issues/56477))

> - **Fixed** incorrect UID and GID assignment for `istio-proxy` and `istio-validation` containers on OpenShift when TPROXY mode is enabled.

Author: ```suggestion
- **Fixed** incorrect UID and GID assignment for `istio-proxy` and `istio-validation` containers on OpenShift when TPROXY mode was enabled.
```
Assuming intent was past tense here as with other suggestions?

  
- **Fixed** ignoring the `topology.istio.io/network` label on the system namespace when `discoverySelectors` are in use.
  ([Issue #56687](https://github.com/istio/istio/issues/56687))

> - **Fixed** an issue where access log not being updated when referenced service created later than the Telemetry resource.  ([Issue #56825](https://github.com/istio/istio/issues/56825))

Author: ```suggestion
- **Fixed** an issue where access logs were not being updated when the referenced service was created later than the Telemetry resource.  ([Issue #56825](https://github.com/istio/istio/issues/56825))
```
